process-automation/latest/model/authentication.md

```diff
@@ -43,3 +45,41 @@ To create authentication:
 The following is an authentication created using Basic authentication and a username and password.
 
 ![Authentication]({% link process-automation/images/authentication.png %})
+
+## Upgrade authentication for deployed application
+
+You can upgrade the authentication of your application using the same version of your application. You can also upgrade the authentication of your application and change the deployed version.
+
+To upgrade your authentication:
+
+1. Expand the **Devops** section on the left of the Admin App and then select **Application Instances**.
+
+2. For the application you want to upgrade, click the three dots on the right of the last column and then select **Upgrade**.
+
+3. Select the version you want to upgrade to from the **Upgrade** dropdown list.
+
+   If you select a different version of the application to the currently deployed one you will be required to enter authentication details further in this process. You can see the currently deployed version from the **Version** column on the **Application Instances** window.
```

Comment:
If you select different version of the application, but selected application has exactly (identical) the same authentications (with all parameters, descriptions, etc.) as currently deployed version, it's optional to enter new authentication values by clicking on the checkbox (you can still use the currently deployed ones if checkbox is unchecked)

Reply:
Updated